packages/amplify_storage_s3/example/lib/main.dart

```diff
@@ -121,7 +121,7 @@ class _MyAppState extends State<MyApp> {
       print('In list');
       S3ListOptions options =
           S3ListOptions(accessLevel: StorageAccessLevel.guest);
-      ListResult result = await Amplify.Storage.list(options: options);
+      ListResult result = await Amplify.Storage.list(path: "public", options: options);
```

Comment:
Is this needed?

Reply:
Sure, we can revert this.

Reply:
Only if it works, I havent tested recently. If this is needed we need to update the library or the docs.